python/paddle/fluid/executor.py

```diff
@@ -452,7 +454,9 @@ def run(self,
             fetch_var_name='fetch',
             scope=None,
             return_numpy=True,
-            use_program_cache=False):
+            use_program_cache=False,
+            infer_inputs=None,
+            infer_attrs=None):
```

Comment:
what is this for? If this is config, it should be put in with_inference_optimize.

Reply:
config is used to create paddle predictor, `infer_attrs` is inference attribute, etc. `batch_size`, `wormup`.

Reply:
warmup is contrib? batch size can be found in `feed`?

Reply:
warmup is not an option of the existing AnalysisConfig.

warmup is just a configuration for profiling, not for inference.

We port the inference to python just for convenient of profiling and testing, so warm up is important.